src/api/app/models/concerns/scm_sync_enabled_step.rb

```diff
@@ -1,13 +1,22 @@
+require 'addressable'
```

Comment:
This line isn't needed, it should be removed.

Reply:
Done